database.sql

```diff
@@ -0,0 +1,115 @@
+CREATE TABLE Brands (
```

Comment:
all tables can start with small letter because they will be translated to it anyway